tests/visualize_test.py

```diff
@@ -417,7 +417,7 @@ def test_heatmap_long_square__grid(long_grid_archive):
                   extensions=["png"])
 def test_heatmap_long_square__cvt(long_cvt_archive):
     plt.figure(figsize=(8, 6))
-    cvt_archive_heatmap(long_cvt_archive, square=True)
+    cvt_archive_heatmap(long_cvt_archive, aspect="equal")
```

Comment:
This seems to be wrong? See the `cvt_archive_heatmap_long_square.png` file -- the images look very different.

Reply:
It turns out you forgot to take out the `if square` in `cvt_archive_heatmap` -- make sure to fix that.